src/main/java/Task.java

```diff
@@ -0,0 +1,89 @@
+public class Task {
+
+    private static int numberOfTasks = 0;
+
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        setNumberOfTasks();
+        printNewTaskAddedMessage();
+    }
+
+    public Task() {
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "V" : " ");  // mark done task with V
+    }
+
+    public static int getNumberOfTasks() {
+        return numberOfTasks;
+    }
+
+    public static void setNumberOfTasks() {
+        Task.numberOfTasks++;
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+        printMarkAsDoneMessage();
+    }
+
+    protected void printNewTaskAddedMessage() {
+        drawDivider();
+        System.out.println("Karlett now remembers:\n" + "  " + this.toString());
```

Comment:
Perhaps the method calling of "this.toString()" could be just "this"?

Reply:
Thank you for your suggestions! I have made improvements accordingly :)